modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -44,6 +44,9 @@ package object dao extends MoreTupleOps with ToUserProgramRoleOps {
   implicit val InstantMeta: Meta[Instant] =
     Meta[Timestamp].nxmap(_.toInstant, Timestamp.from)
 
+  implicit val DurationMeta: Meta[Duration] =
```

Comment:
We use `java.time.Instant` elsewhere and `java.time.Duration` in the F2 config, fwiw.